README.md

```diff
@@ -78,7 +90,7 @@ Rails 3.x support is likely to be dropped in the next major version.
 The gem version is specified in the Rakefile.
 
 ### Contributing to activerecord_autoreplica
- 
+
```

Comment:
I'd recommend fixing your diff to remove all of these whitespace changes. You're going to make the upstream maintainer very unhappy when reviewing this otherwise.